src/app/content/highlights/reducer.ts

```diff
@@ -41,6 +43,11 @@ const reducer: Reducer<State, AnyAction> = (state = initialState, action) => {
     case getType(locationChange): {
       // Noops for locationChange dispatched when search query changes
       if (action.payload.action === 'REPLACE') { return state; }
+
+      const summaryShouldBeOpen =
+        getParamFromQuery(modalQueryParameterName, action.payload.location.search) === modalUrlName
```

Comment:
i would support adding the parsed search query to the payload, we already add it to state, and it would simplify all these reducers

Reply:
this is still outstanding